keyboards/levinson/keymaps/jyh/keymap.c

```diff
@@ -0,0 +1,232 @@
+				[_QWERTY] = KEYMAP(
+								HPR_TAB, KC_Q   , KC_W   , KC_E   , KC_R , KC_T   ,    KC_Y   , KC_U , KC_I   , KC_O   , KC_P   , KC_BSPC,
+								CTL_ESC, KC_A   , KC_S   , KC_D   , KC_F , KC_G   ,    KC_H   , KC_J , KC_K   , KC_L   , KC_SCLN, CTL_QOT,
+								SFT_MIN, KC_Z   , KC_X   , KC_C   , KC_V , KC_B   ,    KC_N   , KC_M , KC_COMM, KC_DOT , KC_SLSH, SFT_ENT,
+								NUMPAD , KC_MEH , KC_LGUI, KC_LALT, LOWER, CTL_SPC,    KC_SPC, RAISE, KC_RALT, KC_RGUI, KC_HYPR, MO(_FUNC)
+								),
+
+
+/* Raise
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * |  `   |  1   |  2   |  3   |  4   |  5   |    |  6   |  7   |  8   |  9   |  0   |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      | Left |  Up  | Down | Right|  (   |    |  )   |  -   |  =   |  [   |  ]   |  |   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |  ;   |  '   |  *   |  ,   |  {   |    |  }   |  _   |  +   |  .   |  /   |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+				[_RAISE] = KC_KEYMAP(
+								HPR_GRV, 1  , 2  , 3  , 4  , 5  ,      6  , 7  , 8  , 9  , 0  ,       ,
+               ,LEFT, UP ,DOWN,RGHT,LPRN,     RPRN,MINS, EQL,LBRC,RBRC,CTL_PIP,
+								       ,COLN,QUOT,ASTR,COMM,LCBR,     RCBR,UNDS,PLUS,    ,    ,       ,
+								       ,    ,    ,    ,    ,    ,         ,    ,    ,    ,    ,
+								),
+
+/* Lower
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * |  ~   |  !   |  @   |  #   |  $   |  %   |    |  ^   |  &   |  *   |  (   |  )   |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Del  | Vol+ | Mute | Play | PgUp |  :   |    |  '   |  -   |  =   |  [   |  ]   |  \   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      | Vol- | < M  |  M > | PgDn |  ;   |    |  "   |  |   |  ,   |  .   |  /   |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+				[_LOWER] = KC_KEYMAP(
+								HPR_TIL,EXLM, AT ,HASH,DLR , PERC  ,     CIRC,AMPR,ASTR,LPRN,RPRN,       ,
+								CTL_DEL,VOLU,MUTE,MPLY,PGUP, COLN  ,     QUOT,MINS, EQL,LBRC,RBRC,CTL_BSL,
+								       ,VOLD,MPRV,MNXT,PGDN, SCLN  ,     DQUO,PIPE,COMM, DOT,SLSH,       ,
+								       ,    ,    ,    ,    ,       ,         ,    ,    ,    ,    ,
+								),
+
+				[_FUNC] = KC_KEYMAP(
```

Comment:
`KC_KEYMAP` is no longer used for this keyboard

Please replace this with `LAYOUT_kc` or add KC_ to all of the keys and use  `LAYOUT_ortho_4x12`

```suggestion
				[_FUNC] = LAYOUT_kc(
```

```suggestion
				[_FUNC] = LAYOUT_ortho_4x12(
```